be/src/exec/olap_scanner.cpp

```diff
@@ -86,6 +88,14 @@ Status OlapScanner::prepare(
             LOG(WARNING) << ss.str();
             return Status::InternalError(ss.str());
         }
+        _tablet_schema = _tablet->tablet_schema();
```

Comment:
OlapScanner is not used in vectorized engine.

Reply:
VOlapScanner is modified too.